canonical/react-components
# feat: add application layout side navigation components

## Done

- Add SideNavigation and child components. These components are used by the application layout but can also be a used in other Vanilla layouts.

## QA

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```

### QA steps

- Open the demo.
- Scroll down to the the SideNavigation docs and check they look OK.
